packages/apollo/src/index.ts

```diff
@@ -16,13 +17,15 @@ export { onError, HttpLink, Mutation, Query, Provider };
 
 export type Settings = {
   links?: ApolloLink[];
-  resolvers?: Resolvers;
+  resolvers?: Resolvers | Resolvers[];
+  typeDefs?: string | string[] | DocumentNode | DocumentNode[];
```

Comment:
`typeDefs?: ApolloClientOptions['typeDefs']`

Reply:
`ApolloClientOptions` requires 1 type argument. Looking at Apollo, it's `ApolloClientOptions<TCacheShape>`, but I'm not sure where to import `TCacheShape` from. 🤔

Reply:
@miles Updated! I'm passing <null> as the value for `TCacheShape`, since we're not using cache here and the parameter is required.